ui/src/app/store/subnet/types/enum.ts

```diff
@@ -6,14 +6,6 @@ export enum IPAddressType {
   DISCOVERED = 6,
 }
 
-export enum IPAddressTypeLabel {
-  AUTO = "Automatic",
-  STICKY = "Sticky",
-  USER_RESERVED = "User reserved",
-  DHCP = "DHCP",
-  DISCOVERED = "Discovered",
-}
-
```

Comment:
@andesol Instead of using a second enum for getting the display text, I've used a util instead. This is a bit of a preference thing, but enums are pretty awkward to use since you can't use computed values for their keys. i.e. it would be good if we could do:
``` ts
export enum IPAddressTypeLabel {
  [IPAddressType.AUTO] = "Automatic",
  [IPAddressType.STICKY] = "Sticky",
  ...
}
```

But since we can't, I generally prefer to use [const assertions](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html#const-assertions) or in this case just a separate util.